health_check/mixins.py

```diff
@@ -33,8 +33,9 @@ def _run(plugin):
             try:
                 return plugin
             finally:
-                from django.db import connection
-                connection.close()
+                from django.db import connections
+                for connection in connections.all():
+                    connection.close()
```

Comment:
There is actually a method on the `ConnectionHandler` that is very similar, but also add some exception handling for already closed connections.
```suggestion
                from django.db import connections
                connections.close_all()
```